django/db/migrations/questioner.py

```diff
@@ -188,6 +192,24 @@ def ask_rename(self, model_name, old_name, new_name, field_instance):
         return self._boolean_input(msg % (model_name, old_name, model_name, new_name,
                                           field_instance.__class__.__name__), False)
 
+
+    def ask_small_to_big_autofield(self, field_name, model_name):
+        """Adding an auto_now_add field to a model."""
```

Comment:
@shaib nice catch — hard to focus on details during the sprints — much appreciated.